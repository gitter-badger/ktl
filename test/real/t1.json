    {
        "name": "John Doe",
        "colors": [
            "red",
            "green",
            "blue",
            "yellow",
            "orange",
            "pink",
            "black",
            "white",
            "beige",
            "brown",
            "cyan",
            "magenta"
        ]
    }