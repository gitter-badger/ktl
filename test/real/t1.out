Hello John Doe!


<ul>
    
    <li class="color">red</li>
    
    <li class="color">green</li>
    
    <li class="color">blue</li>
    
    <li class="color">yellow</li>
    
    <li class="color">orange</li>
    
    <li class="color">pink</li>
    
    <li class="color">black</li>
    
    <li class="color">white</li>
    
    <li class="color">beige</li>
    
    <li class="color">brown</li>
    
    <li class="color">cyan</li>
    
    <li class="color">magenta</li>
    
</ul>
